src/lib/Form.svelte

```diff
@@ -0,0 +1,252 @@
+
+        <label class="form-label" for="Bedrijfsnaam">
+
+            <span class="star">Bedrijfsnaam</span>
+            <input aria-required="true"
+                   name="Bedrijfsnaam"
+                   id="Bedrijfsnaam"
+
+                   required type="text"/>
+        </label>
+        <label class="form-label " for="Bedrijfswebsite">
+
+            <span class="star">Bedrijfswebsite</span>
+            <input aria-required="true"
+                   name="Bedrijfswebsite"
+                   id="Bedrijfswebsite"
+                   required type="text"/>
+        </label>
+
+
+        <label class="form-label input-textfield" for="ik-wil-graag-lid-worden-omdat">
+            <span class="star">Ik wil graag lid worden omdat </span>
+            <textarea required aria-required="true" cols="5" placeholder="" name="Textarea" id="Textarea" rows="1"></textarea>
+        </label>
+
+        <label class="dropown" for="FTE">
+            <span class="star">FTE</span>
+            <select aria-required="true" bind:value={selectedValue}
+                    name="FTE"
+                    id="FTE"
+
+                    required>
+
+                <option value="">
+                </option>
+                <option value="">
+                    1-9
+                </option>
+                <option value="10-24">
+                    10-24
+                </option>
+                <option value="25-49">
+                    25-49
+                </option>
+                <option value="50-99">
+                    50-99
+                </option>
+                <option value="100-249">
+                    100-249
+                </option>
+                <option value="250+">
+                    250
+                </option>
+            </select>
+
+        </label>
+
+        <TextButton/>
+    </fieldset>
+</form>
+
+<style>
+
+    form {
+        color: var(--grey);
+    }
+
+    fieldset {
+        border: none;
+    }
+
+    legend {
+        position: absolute;
+        top: 0;
+        left: -9999px;
+    }
+
+    label {
+        margin-top: 1.8rem;
+        display: grid;
+        width: 100%;
+        border-bottom: 1.5px solid var(--black);
+        gap: 0.8rem;
+    }
+
+    span {
+        grid-row: 1;
+        grid-column: 1;
+        height: 2rem;
+
+    }
+
+    input {
+        grid-row: 2;
+        grid-column-start: 1;
+        grid-column-end: 3;
+    }
+
+    input, textarea, select {
+        border: none;
+    }
+
+
+    textarea {
+        height: auto;
+    }
+
+    .star:after {
+        content: "*";
+        font-size: 2rem;
+        color: red;
+
+    }
+
+    .no-star:after {
+        font-size: 2rem;
+
+        content: "";
+    }
+
+
+    /*ui states*/
+    input:focus, textarea:focus, select:focus, option:focus {
+        outline: 2px solid var(--blue);
+    }
+
+    textarea {
+        width: 100%; /* Make it full width */
+    }
+    /* form focus valid  */
+
+
+    /* form focus valid  */
```

Comment:
Ik zie hier twee keer form focus valid staan deze kunnen weg of 1tje laten staan